packages/elements/src/react/sign-in/index.ts

```diff
@@ -14,6 +14,12 @@ export {
   SignInVerification as Verification,
 } from './verifications';
 
+export {
+  SignInName as Name,
```

Comment:
Do we need `Name`? Is it used somewhere in our existing flow?

Reply:
Felt like a nicety given it's use in `Salutation`, such that it doesn't fall back to a phone number or email.

Reply:
I'd say let's remove for now unless we get feedback asking for it specifically. My main signal is that we're not using it in the fixture app.